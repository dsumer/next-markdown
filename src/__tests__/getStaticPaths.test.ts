import NextMarkdown from '..';

test('generate nextmd from a file tree', async () => {
  const nextMD = NextMarkdown({ pathToContent: 'src/__tests__/__filesystem__/' });

  expect(await nextMD.getStaticPaths()).toMatchSnapshot();
});
