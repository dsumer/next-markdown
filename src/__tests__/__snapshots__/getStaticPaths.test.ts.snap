// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generate nextmd from a file tree 1`] = `
Object {
  "fallback": false,
  "paths": Array [
    Object {
      "params": Object {
        "nextmd": Array [
          "about",
        ],
      },
    },
    Object {
      "params": Object {
        "nextmd": Array [
          "b",
        ],
      },
    },
    Object {
      "params": Object {
        "nextmd": Array [
          "blog",
        ],
      },
    },
    Object {
      "params": Object {
        "nextmd": Array [
          "blog",
          "post-1",
        ],
      },
    },
    Object {
      "params": Object {
        "nextmd": Array [
          "blog",
          "post-2",
        ],
      },
    },
    Object {
      "params": Object {
        "nextmd": Array [
          "blog",
          "post-3",
        ],
      },
    },
    Object {
      "params": Object {
        "nextmd": Array [
          "path",
          "c",
        ],
      },
    },
    Object {
      "params": Object {
        "nextmd": Array [
          "path",
          "to",
          "e",
        ],
      },
    },
    Object {
      "params": Object {
        "nextmd": Array [
          "path",
          "to",
          "f",
        ],
      },
    },
  ],
}
`;
